kit/themes/src/Colors.ts

```diff
@@ -92,6 +93,7 @@ const StaticTheme = {
     [ColorVariants.StaticBackgroundNegative]: 'rgba(231, 23, 23, 0.12)',
     [ColorVariants.StaticBackgroundPositive]: 'rgba(54, 192, 92, 0.12)',
     [ColorVariants.StaticBackgroundOverlay]: 'rgba(4, 4, 4, 0.4)',
+    [ColorVariants.StaticBackgroundSurfPrimary]: '#4963E6',
```

Comment:
@savelichalex @vetkamak what do you think to keep this color only in Surf for now? (i.e. just export Pressable)